README.md

```diff
@@ -0,0 +1,58 @@
+![Screenshot of 'Year Data' cells](resources/readme/tut5.png).
+
+### Obtaining and readying senior portraits of students
+
+1. Email the GMHS registrar, Lynette Kemp, for a list of students in your grade. The registrar is the administrator who manages student enrollment and will have this information on hand; their contact information is listed in the [GMHS staff directory](https://www.fccps.org/o/gmhs/staff?filter_id=%5B68455%5D).
+2. Email the yearbook club teacher lead and ask for senior portraits for the senior map. They will share a folder with you on Google Drive containing all the portraits.
+   - These images will be in alphabetical order with file names of 00001.jpeg, 00002.jpeg, etc.
+   ![Screenshot of numbered senior portraits](resources/readme/tut6.png)
+3. Download all of the photos from Google Drive into a folder on your desktop.
+4. Download the photo naming script available at [`resources/tools/namePictures.py`](https://github.com/apc-gmhs/seniormap/blob/gh-pages/resources/tools/namePictures.py).
+5. Convert the spreadsheet of names into a 2D array to replace `[['firstname', 'lastname']]`.
+   - For example `[['Bill', 'Axel'], ['Sally', 'Breeze'], ['Manuel','Collins']]`
+   - There's a number of online tools that will do this for you, such as [this site](https://www.seabreezecomputers.com/excel2array).
+6. Run the Python script, renaming all of the unnamed photos to their corresponding student names. 
+   - It's important to note that both the array of names and the student photos must be aligned and set up correctly to function properly. For example if a student is present in the array, but doesn't have a photo, all of the photo's names will be off by one. Setting up the script correctly is a little bit time consuming, but far faster than manually renaming each photo.
+7. Create a new folder inside of `portraits` named with the corresponding year.
+8. Upload the pictures to that folder.
+
+### Adding a new student to the map after form submission 
+
+1. After a student submits the form, a new entry will appear in the the corresponding spreadsheet.
+2. If the student appears on the map, highlight the entry in green to indicate they're visible.
+3. If the student does not appear on the map, the college they are attending must be added .
+   - To do this, create a new entry in 'Shared Data' (Senior Map/Shared Data/Location Information) for both the coordinates and logos subsheets.
+   - Mimicking existing cells, add the coordinates, and the logo to the entries.
+   ![Screenshot of logo cells](resources/readme/tut3.png)
+   ![Screenshot of logo cells](resources/readme/tut4.png)
+   - A good way to see if certain schools don't have logos is by using the developer console. The names of institutions missing logos will appear here.
+   ![Screenshot of logo cells](resources/readme/tut2.png)
```

Comment:
tut2 comes after tut3 and 4—maybe we should rename these images to describe what's actually in them rather than their order in the document, that way there's no numbers to change if we reorder the README.

Reply:
definitely a good idea - I didn't do it linearly so it got messed up.